Cartfile.resolved

```diff
@@ -1,3 +1,3 @@
-github "Quick/Nimble" "v6.1.0"
+github "Quick/Nimble" "v7.0.0"
 github "Quick/Quick" "v1.1.0"
-github "ReactiveX/RxSwift" "3.4"
+github "ReactiveX/RxSwift" "3.5.0"
```

Comment:
Don't enought version `3.4` of RxSwift?
Also, few days ago we just moved from strong `3.x.0` dependency in favor of `3.x`